federation/src/config/index.ts

```diff
@@ -11,7 +11,7 @@ Decimal.set({
 */
 
 const constants = {
```

Comment:
What the difference between:
- gradido/federation
- gradido/backend/src/federation